scripts/pin_the_slow_way.py

```diff
@@ -36,6 +36,7 @@
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libblitz': 'libblitz 0.10|0.10.*',
+          'libevent': 'libevent 2.0.*',
```

Comment:
Based on http://abi-laboratory.pro/tracker/timeline/libevent/ you might want to pin like this: `libevent >=2.0.19,<2.1.*`